sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeFormatter.scala

```diff
@@ -28,14 +28,22 @@ import java.util.regex.Matcher
 object CodeFormatter {
   val commentHolder = """\/\*(.+?)\*\/""".r
 
-  def format(code: CodeAndComment): String = {
+  def format(code: CodeAndComment): String = format(code, -1)
+
+  def format(code: CodeAndComment, maxLines: Int): String = {
```

Comment:
@srowen I can make that change. Is there any need to scaladoc the maxLines param? Or is the function definition easy enough to follow?

```
def format(code: CodeAndComment, maxLines: Int = -1): String
```